application/src/main/java/org/togetherjava/tjbot/commands/CommandHandler.java

```diff
@@ -25,33 +28,35 @@
  * Commands need to be added to the commandList
  */
 public class CommandHandler extends ListenerAdapter {
-    private final static Logger logger = LoggerFactory.getLogger(CommandHandler.class);
+    private static final Logger logger = LoggerFactory.getLogger(CommandHandler.class);
 
     private final List<Command> commandList = new ArrayList<>();
     private final Map<String, Command> commandMap;
 
-    public CommandHandler() {
-        commandList.addAll(List.of(new ReloadCommand(this)));
+    public CommandHandler(Database database) {
+        commandList.addAll(
+                List.of(new ReloadCommand(this), new PingCommand(), new DatabaseCommand(database)
+                // add your command here
+                ));
 
         commandMap = commandList.stream()
             .collect(Collectors.toMap(Command::getCommandName, Function.identity()));
     }
 
     @Override
     public void onReady(@NotNull ReadyEvent event) {
-        event.getJDA().getGuildCache().forEach(guild -> {
-            guild.retrieveCommands().queue(commands -> {
-                boolean hasReloadCommand =
-                        commands.stream().anyMatch(command -> command.getName().equals("reload"));
+        event.getJDA().getGuildCache().forEach(guild -> guild.retrieveCommands().queue(commands -> {
```

Comment:
I intentionally made usage of the brackets here to make it clear it's within a lambda.

Reply:
It was also being considered a code smell by our checks, also seems pretty clear enough

Reply:
First of all, this is a add ping command feature, not a styling PR.
This shouldn't contain any styling changes, should be separated, and second of all.

We put methods on the next line constantly, and this line contains 5 method calls on the same line.
I completely disagree with readability, in comparison to all the other code this is extremely unreadable in my opinion.